content/posts/2023/pdf-root-files-preview.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Introducing PDF and ROOT file previews in REANA web"
+date: 2023-06-26T14:00:00+01:00
+---
+
+We are excited to announce that the REANA web interface now supports previewing of PDF and ROOT files.
+This means that instead of downloading these files to view them, you can simply inspect them right away,
+without leaving your browser!
+
+<!--more-->
+
+For a long time, the REANA web UI has allowed users to preview certain types of files present in the
+workflow workspace. This has proven to be very handy for users who want to quickly glance through
+images and text files in their workspaces without having to download them. We are now extending
+this capability to PDF and ROOT files as well.
+
+## PDF files preview
+
+Starting from REANA 0.9.1, when you click on a PDF file in your workflow's workspace in the web UI, an embedded
+PDF viewer will load the document. You will be able to scroll through pages, zoom in and out,
+or even download the PDF if necessary.
+You can use it to quickly inspect the content of any PDF generated by your analysis or uploaded to your
+workspace.
+Here is a sneak peek of how it looks:
+
+![PDF file preview](/images/pdf-files-preview.png)
+
+## ROOT files preview
+
+ROOT files can also be previewed directly in the web UI.
+We have embedded a ROOT file viewer, based on [JSROOT](https://root.cern/js/), enabling you to inspect your
+ROOT files without leaving your browser.
+You can navigate through the file structure, interact with charts and ROOT objects, zoom on some details,
+and download the file if needed.
+Here you can see the ROOT file viewer in action:
+
+![ROOT file preview](/images/root-files-preview.gif)
+
+These enhancements aim to save time and streamline the process of navigating and inspecting files
+in your REANA workflow workspace.
+We are continually working on improving your experience with REANA, and we hope that
+they will prove beneficial in your research and data analysis.
+If you have any feedback or suggestions, please don't hesitate to reach out.
+
+## See also:
+- [Previewing HTML files since REANA 0.8.1](/posts/2022/reana-0.8.1/#previewing-html-workspace-files-directly)
```

Comment:
We need also to link to https://blog.reana.io/posts/2023/reana-0.9.1/#previewing-pdf-and-root-files

I would call the first item "Previewing HTML files" (since REANA 0.8.1), and the second list item "Previewing PDF and ROOT files" (since REANA 0.9.1).